docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
+3. Responses MUST have the same value for the producedAt and thisUpdate times.
+4. The difference in time between the thisUpdate and nextUpdate fields MUST be greater than or equal to 8 hours and MUST NOT be greater than or equal to 192 hours (7 days).
+5. For OCSP responses directly signed by the issuing CA, the response MUST NOT include any certificates within the BasicOCSPResponse's certs field.
+6. For OCSP responses signed by a Delegated OCSP Responder Certificate, the response MUST only include the Delegated OCSP Responder Certificate within the BasicOCSPResponse's certs field and MUST NOT include any additional certificates.
+
 ### 7.3.1 Version number(s)
 
+OCSP responders MUST support v1 requests, as specified in RFC6960. OCSP responders MUST produce v1 responses, as specified in RFC6960.
+
 ### 7.3.2 OCSP extensions
 
+1. As specified in RFC5019 for pre-produced responses, responses MUST NOT include the id-pkix-ocsp-nonce extension.
```

Comment:
As discussed during the SHA-1 discussions, CAs supporting id-pkix-ocsp-nonce can be induced to sign material used to produce collisions. It also is actively detrimental to stapling.

If there's a use case for CAs to support this, I think we need to specify an upper-bound, and would want to revisit the profile for OCSP Responder Certificates to minimize risk. There's no good answers here.